Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/EditorMeshComponent.cpp

```diff
@@ -253,6 +254,12 @@ namespace AZ
             AzToolsFramework::ToolsApplicationEvents::Bus::Broadcast(
                 &AzToolsFramework::ToolsApplicationEvents::InvalidatePropertyDisplay,
                 AzToolsFramework::Refresh_EntireTree);
+
+            // Register this component with the editor mode feedback system
+            if (auto* editorModeFeedbackInterface = AZ::Interface<EditorModeFeedbackInterface>::Get())
+            {
+                editorModeFeedbackInterface->RegisterDrawableComponent({ GetEntityId(), GetId() }, m_controller.m_meshHandle);
```

Comment:
nit: Might be good to add `EntityComponentIdPair` explicitly here (does making tracking down usages easier)

Reply:
Good catch, fixed 👍